packages/runtime/container-runtime/src/dataStoreContext.ts

```diff
@@ -377,6 +377,13 @@ export abstract class FluidDataStoreContext extends TypedEventEmitter<IFluidData
         const channel = await factory.instantiateDataStore(this, existing);
         assert(channel !== undefined, 0x140 /* "undefined channel on datastore context" */);
         this.bindRuntime(channel);
+
+        if (!existing) {
+            // Load the handle to initialize the object. The only implementation of IFluidDataStoreChannel is
+            // FluidDataStoreRuntime, which exposes its handle.
+            const channelAsFluidObject: IFluidDataStoreChannel & FluidObject<IFluidHandle> = channel;
+            await channelAsFluidObject.IFluidHandle?.get();
```

Comment:
i wouldn't expect any new call to the `IFluidHandle` property the should all transition to `IFluidLoadable`